src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/x64/fuse_reshape_transpose_to_sdpa.cpp

```diff
@@ -204,7 +204,7 @@ class FuseSDPAReshapeTransposeTest : virtual public ov::test::SubgraphBaseTest,
 };
 
 TEST_P(FuseSDPAReshapeTransposeTest, CompareWithRefs) {
-    SKIP_IF_CURRENT_TEST_IS_DISABLED();
+    GTEST_SKIP() << "TODO: investigate perf-regression on ICX." << std::endl;
```

Comment:
Better to use "skip_tests_config.cpp" for such purposes + add jira ticket number.

Reply:
The test is enabled.